blogposts/2022/release-3.40.md

```diff
@@ -0,0 +1,16 @@
+---
+title: 'Sourcegraph 3.40 release'
+publishDate: 2022-05-20T10:00-07:00
+description: 'Sourcegraph 3.40 introduces...'
+tags: [blog, release]
+slug: 'release/3.40'
+published: false
+heroImage: https://storage.googleapis.com/sourcegraph-assets/blog/3.40/sourcegraph-3-40-release.png
+socialImage: https://storage.googleapis.com/sourcegraph-assets/blog/3.40/sourcegraph-3-40-release.png
+changelogItems:
+  - description:
+    url:
+    category:
+---
+
+Sourcegraph 3.40 is now available! Here are some highlights from this release:
```

Comment:
```suggestion
Sourcegraph 3.40 is now available! Here are some highlights from this release:

# Local navigation for the top 8 languages is now faster and more accurate

We [previously improved Java](https://about.sourcegraph.com/blog/release/3.39/#Local-navigation-for-Java-code-is-now-faster-and-more-accurate), and since then we’ve added support for Go, C#, Python, JavaScript, TypeScript, C/C++, and Ruby.
```